docs/source/technical_tutorials/search/searching_the_stac_catalog.ipynb

```diff
@@ -7,7 +7,11 @@
    "source": [
```

Comment:
<ol><li>Typo in "Development Seed"</li><li>This is probably very picky, but I'd probably hyperlink "MAAP STAC catalog" (unless you're purposefully showing the link there)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>